src/style.css

```diff
@@ -1 +1,95 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0 50px;
+}
+
+@media (min-width: 1024px) {
+  .wrapper {
+    width: 1024px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .wrapper {
+    width: 1200px;
+  }
+}
```

Comment:
remove, don't hardcode this width, just add padding from the horizontal sides of the viewport